commit 2007da5f17f4a2f79305488ed361efcaaeb2a6fe
Author: Robert Bogart <robert.bogart@joyent.com>
Date:   2019-09-16T17:54:41+00:00 (3 weeks ago)
    
    MANTA-4552 Rebalancer agent test init automation breaks if /var/tmp/rebalancer does not already exist

diff --git a/test/agent/test_init.sh b/test/agent/test_init.sh
index befe1b7..f16abba 100755
--- a/test/agent/test_init.sh
+++ b/test/agent/test_init.sh
@@ -158,7 +158,7 @@ do
     insert_task "$uuid" "$i"
 
     if [[ "$storage_id" -eq "localhost" ]]; then
-        mkdir "$object_dir"
+        mkdir -p "$object_dir"
         cp "$i" "$object_dir"
     else
         curl -X PUT -T $i "http://$storage_id/$account/$(basename $i)"
